unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/KernelCommunication/RPC/Services/CRDTService/CRDTService.cs

```diff
@@ -0,0 +1,83 @@
+using System;
+using System.Collections.Generic;
+using Cysharp.Threading.Tasks;
+using DCL;
+using DCL.CRDT;
+using Google.Protobuf;
+using KernelCommunication;
+using rpc_csharp;
+using UnityEngine;
+
+namespace RPC.Services
+{
+    public static class CRDTServiceImpl
```

Comment:
This shouldn't be static and it should be moved into the Service Locator

Reply:
It's static because it is stateless, we do not need an instance, it exist just to make it easier to read (because we avoid creating some lambda functions) and creating some static objects to reduce allocations.
We don't need it in service locator since no script needs no access it except for the rpc server to register the service.